classes.js

```diff
@@ -53,250 +51,75 @@ class Garden{
-
-        for (let plant of listOfPlants){
-        description += "\n" + describePlant(plant)
+        for (let plant of this.plants){
+        description += "\n" + plant.describe()
         }
- 
-    return description;
+        return description;
     }
     
-    addPlant = () =>{
-        this.name.push(this.plants);
+    addPlant(plant){
+        this.plants.push(plant);
     }
 }
 
 //Estate
 class Estate{
-    constructor(garden){
+    constructor(){
         this.roseArbor = new Garden("Rose Arbor");
         this.perenialGarden = new Garden("Perennial Garden");
         this.slopePlanters = new Garden("Slope Planters");
     }
 
-    addPlant = () => {
-        if(this.type === "rose"){
-            this.roseArbor.push(plant());
-        
-        } else if (this.isPerennial === true && this.amountOfSunNeeded <= 5){
-            this.perennialGarden.push(plant());
-        
+    addPlant(plant){
+        if(plant.type === "rose"){
+            this.roseArbor.plants.push(plant);
+        } else if (plant.isPerennial && plant.amountOfSunNeeded <= 5) { 
+            this.perennialGarden.plants.push(plant);
         } else {
-            this.slopePlanters.push(plant());
+            this.slopePlanters.plants.push(plant);
         } 
-    
-        return garden.addPlant(plant());
     }
 
-    describe = () => {
-        const estateLength = Object.keys(estate).length;
+    describe(){
+        const estateLength = this.length;
         let description = `The estate has ${estateLength} gardens.`;
-        for (let gardenName in estate){
-            let listofPlants = estate[gardenName];
-            description += "\n" + describeGarden(gardenName, listofPlants);
+        for (let gardenName in this){
+            let garden = this[gardenName];
+            description += "\n" + garden.describe;
         }
     
         return description
     }
 
-    calculateWaterUsagePerWeek = () => {
+    calculateWaterUsagePerWeek (){
         let numGallons = 0;
-        for (let gardenName in estate){
-            let garden = estate[gardenName];
-            for (let plant of garden){
-                numGallons += plant.gallonsWaterPerWeek;
+        for (let gardenName in this){
+            for (let plant of this[gardenName].plants){
+                numGallons += plant.gallonsWaterPerWeek * 10;
             }
         }
-    let newNumGallons = Math.floor(numGallons);
-    return newNumGallons;
+    return numGallons / 10;
     }
 
-    cloneAllTheRosesAndChangeTheirColors = () => {
+    cloneAllTheRosesAndChangeTheirColors () {
         let clonedRoses = [];
-        let estatePlants = Object.keys(estate)
-        
-        for (let i = 0; i < Object.keys(estate).length; i++){
-            let gardenName = Object.values(estate[i]);
     
-            for(let flower in gardenName){
-    
-                let plant = gardenName[flower];
+        for (let gardenName in this){
+            for (let plant of this [gardenName].plants){
```

Comment:
I'm actually surprised this loop functions properly with a space between `this` and `[gardenName]` but just make sure in the future  to not put a space between object properties (just for clean code sake)